packs/data/extinction-curse-bestiary.db/iffdahsil.json

```diff
@@ -96,7 +96,7 @@
                 },
                 "spelldc": {
                     "dc": 42,
-                    "value": 34
+                    "value": 33
```

Comment:
```suggestion
                    "value": 34
```

Reply:
@SpartanCPA Any reason this is set to 34? Statblock doesn't list an attack modifier and the Moderate Attack bonus in GMG for Level 21 creatures is 33.

Reply:
nvm, I see how it was determined now.

Reply:
Per Table 2–11: Spell DC and Spell Attack Bonus, spell attack bonus is always DC - 8